exonum-java-binding/core/rust/ejb-app/TUTORIAL.md

```diff
@@ -9,7 +9,7 @@ of the Contribution Guide.
 You also need a ready-to-use Exonum Java service. You can use 
 [cryptocurrency-demo][cryptocurrency-demo] as an example, and find information about 
 implementing your own Exonum service 
-in the [user guide](https://exonum.com/doc/version/latest/get-started/java-binding/).
+in the [user guide](https://exonum.com/doc/version/0.10/get-started/java-binding/).
```

Comment:
404 for https://exonum.com/doc/version/0.10/get-started/java-binding/ ;(

Reply:
Sure, 0.10 is not posted yet
Will add `[DO-NOT-MERGE]`

Reply:
@MakarovS Have you confirmed the path the docs will be available at?

If/when yes, we will merge into `v0.4-release` branch, not waiting for the publication.

Reply:
Yes, @Maria-Nosyk confirmed that it'd be `0.10`

Reply:
Wonderful, please rebase then onto the release branch (we will sync it with master shortly after release).

Reply:
We want Javadocs to include updated links, right?

Reply:
Sure, that’s why we will publish the updated versions on our website once we fix all issues and make a release.